### PR TITLE
Making onnx opset 13 default

### PR DESCRIPTION
**Pull request by corey-nm**
For torch-1.12, the default onnx opset should be 13. Unclear what the default should be for other versions.

### CODE REVIEW

File: integrations/keras/classification.py [hunk 1]
```diff
@@ -572,7 +572,7 @@ def parse_args():
     export_parser.add_argument(
         "--onnx-opset",
         type=int,
-        default=11,
+        default=13,
         help="The onnx opset to use for export. Default is 11",
```
Comment (rahul-tuli) [inline integrations/keras/classification.py:576]:
```suggestion
        help="The onnx opset to use for export. Default is 13",
```

**Comment by corey-nm** [comment]
> A missed 11 -> 13, LGTM otherwise

Great catch haha 🚀

File: src/sparseml/tensorflow_v1/utils/exporter.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
 
 def default_onnx_opset() -> int:
-    return 9 if onnx.__version__ < "1.6" else 11
+    return 9 if onnx.__version__ < "1.6" else 13
```
Comment (bfineran) [inline src/sparseml/tensorflow_v1/utils/exporter.py:41]: is this a tested pathway? if not would prefer to not change

File: integrations/keras/classification.py [hunk 3]
```diff
@@ -572,8 +572,8 @@ def parse_args():
     export_parser.add_argument(
         "--onnx-opset",
         type=int,
-        default=11,
-        help="The onnx opset to use for export. Default is 11",
+        default=13,
```
Comment (bfineran) [inline integrations/keras/classification.py:575]: let's keep this as is for now unless tested

File: integrations/tensorflow_v1/classification.py [hunk 4]
```diff
@@ -546,8 +546,8 @@ def parse_args():
             par.add_argument(
                 "--onnx-opset",
                 type=int,
-                default=11,
-                help="The onnx opset to use for export. Default is 11",
+                default=13,
```
Comment (bfineran) [inline integrations/tensorflow_v1/classification.py:549]: let's keep this as is for now unless tested

**Review by bfineran — COMMENTED** [review]
the tf and keras onnx exporters rely on third party libraries that may have not been updated in a bit, hesitant on updating without further testing

File: integrations/tensorflow_v1/classification.py (hunk 4 above)
Comment (corey-nm) [reply integrations/tensorflow_v1/classification.py:549]: Will do

**Comment by corey-nm** [comment]
Closing, will re-open a different one

**Pull request closed without merging**